src/pathToVictory/services/officeMatch.service.ts

```diff
@@ -588,17 +706,26 @@ export class OfficeMatchService {
         },
         SlackChannel.botPathToVictoryIssues,
       )
-      this.logger.error('No Response from AI! For', searchValues)
+      this.logger.error(`No Response from AI! For ${String(searchValues)}`)
     }
 
     if (content && content !== '') {
       try {
-        const data = JSON.parse(content)
-        if (data?.matchedLabel && data.matchedLabel !== '') {
-          return data.matchedLabel.replace(/"/g, '')
+        const parsed: unknown = JSON.parse(content)
+        if (
+          parsed &&
+          typeof parsed === 'object' &&
+          'matchedLabel' in parsed &&
+          typeof (parsed as { matchedLabel?: string }).matchedLabel === 'string'
+        ) {
```

Comment:
This is duplicated logic from the `JSON.parse` commented on above as well.

`JSON.parse` _needs_ `try-catch` since it'll `throw` when given a poorly formed JSON string.

And there's _gotta_ be a better way to attack the existential key problem.